### DISCUSSION TITLE
Remix example

### DISCUSSION THREAD

**Discussion by jarimustonen** [awslabs/aws-lambda-web-adapter#399] [General]
Would it be possible to have an example for Remix? Also, it would be great if the example would use Vite as Remix is in the process of migrating to Vite.

Remix is gaining quite a lot of traction, so adding examples for it would really make sense.

**Comment by bnusunny** [comment]
This is a good first issue. PR welcome.

**Comment by jarimustonen** [comment]
@thomasfr (who works at AWS) mentioned in [this discussion](https://github.com/remix-run/remix/discussions/4678) that this is being used internally at AWS so maybe that would act as a good starting point for an example?

**Comment by dhstevev** [comment]
bump! Would love to see an example.

**Comment by meza** [comment]
Would love to expedite this if possible. This is my only hope :D

**Reply by bnusunny** [reply]
This is definitely possible. I will add an example soon.

**Comment by bnusunny** [comment]
Remix examples are added: 

https://github.com/awslabs/aws-lambda-web-adapter/tree/main/examples/remix
https://github.com/awslabs/aws-lambda-web-adapter/tree/main/examples/remix-zip
